bukuserver/filters.py

```diff
@@ -57,16 +58,16 @@ def bottom_x_func(query, value, index):
 
 class FilterType(Enum):
 
-    EQUAL = {'func': equal_func, 'text':'equals'}
-    NOT_EQUAL = {'func': not_equal_func, 'text':'not equal'}
-    CONTAINS = {'func': contains_func, 'text':'contains'}
-    NOT_CONTAINS = {'func': not_contains_func, 'text':'not contains'}
-    GREATER = {'func': greater_func, 'text':'greater than'}
-    SMALLER = {'func': smaller_func, 'text':'smaller than'}
-    IN_LIST = {'func': in_list_func, 'text':'in list'}
-    NOT_IN_LIST = {'func': not_in_list_func, 'text':'not in list'}
-    TOP_X = {'func': top_x_func, 'text': 'top x'}
-    BOTTOM_X = {'func': bottom_x_func, 'text': 'bottom x'}
+    EQUAL = {'func': equal_func, 'text': _l('equals')}
+    NOT_EQUAL = {'func': not_equal_func, 'text': _l('not equals')}
+    CONTAINS = {'func': contains_func, 'text': _l('contains')}
+    NOT_CONTAINS = {'func': not_contains_func, 'text': _l('not contains')}
+    GREATER = {'func': greater_func, 'text': _l('greater than')}
+    SMALLER = {'func': smaller_func, 'text': _l('smaller than')}
+    IN_LIST = {'func': in_list_func, 'text': _l('in list')}
+    NOT_IN_LIST = {'func': not_in_list_func, 'text': _l('not in list')}
+    TOP_X = {'func': top_x_func, 'text': _l('top X')}
+    BOTTOM_X = {'func': bottom_x_func, 'text': _l('bottom X')}
```

Comment:
`gettext()`/`_()` requires Jinja context at the time of invocation, so I'm using `lazy_gettext()`/`_l()` here instead